roles/kube-template-cni/templates/multus-crd.yaml.j2

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: flannel
+  namespace: kube-system
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-multus-cfg
+  namespace: kube-system
+  labels:
+    tier: node
+    app: multus
+data:
+  cni-conf.json: |
+    {
+      "name": "minion-cni-network",
+      "type": "multus",
+      "kubeconfig": "/etc/kubernetes/kubelet.conf"
+    }
+  net-conf.json: |
+    {
+      "Network": "{{ pod_network_cidr }}/16",
```

Comment:
your variable is confusing here... you have a trailing `/16`. Is that intentional?

Reply:
it's basically the flannel default, cause this is an embedded flannel portion. I'm just going to leave it for now until someone needs a different network range (with overlapping ip address issues hitting the radar that might come up in the near-ish future)